about.html

```diff
@@ -0,0 +1,196 @@
+          The logo of CC Global Summit 2023 was decided through the logo
+          competition from 8.Jan to 7.Feb
+        </p>
+        <div class="abt-logo-display">
+          <img
+            class="about-logo-img"
+            src="./images/cc-logo.png"
+            alt="The conference logo"
+          />
+        </div>
+      </div>
+      <!--order 1-->
+      <hr class="abt-section-wide-hr" />
+    </section>
+
+    <!--past conference -->
+    <section class="abt-past-confs blur">
+        <!--order 0-->
+            <h2 class="abt-confs-heading">
+                See the past CC Global Summits
+            </h2>
+             <!--order 1-->
+            <hr class="section-heading-hr abt-past-cc-hr" />
+             <!--order 2-->
+            <p class="abt-confs-info">
+                Take a look at the last two CC Global Summits which took place in
+                Buenos Aires and in Warsaw
+            </p>
+           <!--order 3-->
+           <div class="past-conferences">
+            <!-- Past Confs Bootstrap overlay-link1" 2013 -->
+            <a href="#">
+            <div class="container abt-confs-display">
+              <div class="row abt-past-confs-row">
+                <div class="col-md-4 abt-past-confs-col">
+                  <div class="single-content">
+                      <img
+                      class="abt-confs-img cc-2013"
+                      src="./images/cc-2013-img-resized.png"
+                      alt="2013 conference info link"
+                      />
+                      <div class="text-content">
+                        <h4>2013</h4>
+                        <p>CC Global Summit 2013 in Buenos Aires</p>
+                      </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            </a>
+
+            <!-- Past Confs Bootstrap overlay-link2" 2011 -->
+            <!--order 4-->
+            <a href="#">
+            <div class="container abt-confs-display">
+              <div class="row abt-past-confs-row">
+                <div class="col-md-4 abt-past-confs-col">
+                  <div class="single-content">
+                    <img
+                    class="abt-confs-img cc-2013"
+                    src="./images/cc-2011-img-resized.png"
+                    alt="2011 conference info link"
+                    />
+                    <div class="text-content">
+                      <h4>2011</h4>
+                      <p>CC Global Summit 2011 in Warsaw</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            </a>
+            </div>
+    </section>
+    <!--partners section-->
+    <section id="abt-conf-partners" class="partners-section blur">
+      <div class="section-heading dark-bg">
+          <h2 class="section-head">
+              Partners
+          </h2>
+          <hr class="section-heading-hr partners-head-hr" />
+      </div>
```

Comment:
[REQUIRED]We don't have a partners section in the desktop screen size for the About page. Kindly adjust your code to fulfil this requirement.

Reply:
Thanks for the reminder